packages/core/docs/performance/other-optimizations.md

```diff
@@ -0,0 +1,123 @@
+# Other optimizations
+
+There are plenty of general optimizations that didn't fit the previous categories or improve multiple areas of the application.
+
+## Avoid render-blocking resources :orange_book:
+
+Render-blocking resources are scripts, stylesheets, and other imports in the `<head>` that delay the browser from rendering page content to the screen until they are downloaded and parsed. Such resources delay the First Paint - time needed for the browser to render something (i.e., background colors, borders, text, or images) for the first time. To prevent that:
+
+* add the `defer` or `async` attribute to the `<script>`,
+* use `disabled` or `media` attributes for the stylesheets.
+
+Nuxt is deferring every bundle script it generates. If you want to deffer third party scripts, just add them in the `nuxt.config` as shown in the example below:
+
+```javascript
+// nuxt.config.js
+
+export default {
+  head: {
+    script: [
+      {
+        src: `<SCRIPT_URL>`,
+        defer: true
+      }
+    ]
+  }
+};
+```
+
+Additionally, you can add use [media-dependent](https://developer.mozilla.org/en-US/docs/Web/CSS/@import) import in your CSS:
+
+```css
+//main.scss
+@import '@/assets/main.scss' print;
+```
+
+## Inline critical assets :ledger:
+
+Consider delivering critical JavaScript/CSS inline and deferring all non-critical assets. You can reduce the size of your pages by only shipping the code and styles that you need at the given time.
```

Comment:
it shouldn't be a manual work, please suggest some automated way ;)

Reply:
For now there is no easy way to achieve this automatically, should I remove this part, and add it when we will find solution ?